examples/blog/next.config.mjs

```diff
@@ -20,7 +20,11 @@ export default withNextra({
     // development repository.
     config.resolve = {
       ...config.resolve,
-      symlinks: false
+      alias: {
```

Comment:
At the top the comment needs to be adapted, since you no longer use `symlinks: false` (for other examples also)